acceptance_modelisation/grid3d_acceptance_map_creator.py

```diff
@@ -253,7 +256,8 @@ def create_acceptance_map(self, observations: Observations) -> Background3D:
 
         return acceptance_map
 
-    def _create_base_computation_map(self, observations: Observations) -> Tuple[WcsNDMap, WcsNDMap, WcsNDMap, u.Quantity]:
+    def _create_base_computation_map(self, observations: Observations) -> Tuple[
+        WcsNDMap, WcsNDMap, WcsNDMap, u.Quantity]:
```

Comment:
My only comment is on the output. I would either update the declared output to ndarray or change it to make it a WcsNDMap, likely the first is the easieast.